app/design/frontend/base/default/template/esatisfaction/header.phtml

```diff
@@ -5,9 +5,24 @@ if ($status) {
     $applicationId = $helperData->getApplicationId();
     ?>
     <script>
-        window.esat_config =
-            {
-                application_id: '<?php echo $applicationId; ?>'
+        (function (win, doc, application_id, jq, collection) {
+            // Define e-satisfaction collection configuration
+            win.esat_config = {application_id: application_id, collection: collection || {}};
+
+            // Update metadata
+            win.Esat = win.Esat || {};
+            win.Esat.updateMetadata = function (questionnaireId, metadata) {
+                win.esat_config.collection[questionnaireId] = win.esat_config.collection[questionnaireId] || {};
+                win.esat_config.collection[questionnaireId].metadata = metadata;
             };
+
+            // Setup script
+            doc.addEventListener('DOMContentLoaded', function () {
+                var body = doc.getElementsByTagName('body')[0], script = doc.createElement('script');
+                script.async = true;
+                script.src = 'https://collection.e-satisfaction.com/dist/js/integration' + (!!jq ? '' : '.jq') + '.min.js';
+                body.appendChild(script);
+            });
+        })(window, document, "<?php echo $applicationId; ?>", "<?php $helperData->getIsJQueryEnabled() ? true : false ?>", {});

```

Comment:
I think it should be the other way around with the `true` `false` parameters.

Reply:
The preview check was:
`$jqueryInstalled = $helperData->getIsJQueryEnabled();`	
`$fileName = $jqueryInstalled ? 'integration.jq.min.js' : 'integration.min.js';`

Reply:
We have to check the plugin's admin panel to see what's the use of the variable.
The same should apply to the 2.2 version as well.

Reply:
From magento admin panel, helper text about this variable is:

> Select yes if jquery is installed in your magento theme installation

So, let's change the code to behave accordingly.